### DISCUSSION TITLE
recommendation to check the availability of a router

### DISCUSSION THREAD

**Discussion by pipiche38** [zigpy/zigpy#1402] [Q&A] [answered]
Just wonder if you have any best practice on how to check if a router is still available.

Using a read attribute on cluster 0x0000

Using a Network LQI Request or anything else ?

Just wonder from a zigbee perspective what is the lightest way

**Comment by puddly** [comment, marked as answer]
I would rely on `last_seen` (it updates every time a packet is received from the device) and poll a `Basic` attribute if it's been too long.
